<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string notr="true" >border-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="btn2" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnq" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btno" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>确 定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnt" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>退 格</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
